vendor/github.com/IBM-Cloud/power-go-client/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
nit: Please use imperative mood for this comment subject https://github.com/openshift/hypershift/blob/main/docs/content/contribute/index.md instead of "Adding"

Reply:
Renamed the commit